src/pkg/storage/storage.go

```diff
@@ -168,7 +169,16 @@ func mustMatchConfig(
 		}
 
 		vv := cast.ToString(g.Get(tomlK))
-		if v != vv {
+
+		areEqual := false
+
+		if path.DoesPathExistInOS(v) && path.DoesPathExistInOS(vv) {
```

Comment:
Could you please add a comment to describe this code addition? Also once concern I have is that there are non path related keys in the map. This code seems to be doing `os.stat()` call for  all the keys. One could argue that these could be valid file paths on a user's system, could be interpreted incorrectly as a snooping behavior.

Is there a way we could only do this check for file system path key? Btw, thanks for fixing the trailing slash bug! Good find!

Reply:
@pandeyabs please check the current change. Do you concur with the current implementation?